plugins/pitchfork.py

```diff
@@ -0,0 +1,35 @@
+import re
+import string
+
+from errbot import BotPlugin, botcmd
+
+
+class Pitchfork(BotPlugin):
+    """
+    To pitchfork users down to ...
+    """
+
+    @botcmd
+    def pitchfork(self, msg, arg):
+        """
+        To pitchfork user down to ...
+        """
+        match = re.match(r'@?([\w-]+)(?:\s+(?:down\s+)?to\s+(.+))?$',
+                         arg)
+        if match:
+            user = match.group(1)
+            place = match.group(2) if match.group(2) else 'offtopic'
+            return (
+                string.Template("""
```

Comment:
I find unindented multi line strings disrupt the flow when reading a function/file. You could use [dedent](https://docs.python.org/3.6/library/textwrap.html#textwrap.dedent) to have the literal be properly indented and still not have the indentation in the actual string.

However then you'll probably run into the line length limit.... Hmm.... maybe make the pitchfork a bit shorter? I did notice that on mobile I just saw the stick: 
![screenshot_20170726-083940](https://user-images.githubusercontent.com/1924002/28607878-16dab980-71de-11e7-9380-aca4f2a77ed3.png)